data/remnant missions.txt

```diff
@@ -60,27 +62,94 @@ mission "First Contact: Remnant"
+			choice
+				`(Provide the blood sample.)`
+					goto SecondChanceComply
+				`(Refuse and run for your ship.)`
+					goto SecondChanceRefuse
+			label SecondChanceRefuse
+			`	As you turn to start running back to your ship you notice a quick hand sign from the prefect and the soldiers nearby begin raising their weapons and firing at you. As you feel your body begin to liquify under the assault of the strange energy bolts, the last thing you see is the look of pity, resignation, and confusion on the face of the Prefect. Probably wondering why you would decide to return if you were just going to run again. You suppose they'll never know...`
+			die
+			label SecondChanceComply
+			apply
+				"Remnant Blood Test Pure" = 1
+			`	Several of them gather around the one with the scanner as if waiting to see what the results will be. As you watch them you notice that all of them, even the women, have their hair cropped short. Also, their clothing looks like nothing that has ever been in fashion at any point in human history that you know of.`
+			`	Finally the scanner beeps, and a yellow light flashes. You can almost feel the tension leaving the air as the crowd relaxes. "So you are pure. You are human," chants one of the women. "You are one of us."`
+			choice
+				`	"Of course I'm human. But what are human beings doing out here?"`
+				`	"Who are you?"`
+
+			`	Several of them burst into song, singing together in harmony, "We are the hidden ones. We are the Remnant. In time of great chaos to this place we came, lest all true-born humans be killed or enslaved, by those who evolved not, but unwisely were made."`
+			`	As they continue to sing, you begin to piece together their story. They are the descendants of people who fled to this region of space through an unstable wormhole at the peak of the first Alpha War, which means they have been here for more than half a millennium. That doesn't explain why their culture is so radically different, though.`
+			`	Eventually they invite you to a dinner. Their food has strange and complex flavors, not all of them pleasant, but you do your best to be polite and pretend to enjoy it all. After the meal, they entertain you by singing songs, some of which you recognize as classical Earth music. It appears that people in this culture use sign language for day to day communication, and use their voices only for making music.`
```

Comment:
I love this addition. As far as i know, it's never mentioned anywhere, but given the variety of human space (and the space beyond), the player likely had to cope with a large variety food before (even though it's sort of a joke, the news file tells us that some street vendors are selling rat meat, something you'd never see on a paradise world). I like that the player is versed enough to not only not throw up, but even plays their role all the way through